src/DependencyInjection/Configuration.php

```diff
@@ -36,17 +41,7 @@ private function addServersSection($rootNode)
                     ->useAttributeAsKey('alias', false)
                     ->prototype('array')
                         ->children()
-                            ->scalarNode('address')
-                                ->isRequired()
-                                ->validate()
-                                    ->ifTrue(
-                                        function ($v) {
-                                            return substr($v, 0, 6) != 'udp://';
-                                        }
-                                    )
-                                    ->thenInvalid("address parameter should begin with 'udp://'")
-                                ->end()
-                            ->end()
+                            ->scalarNode('address')->isRequired()->end()
```

Comment:
I'm not sure why you've changed this validation, but it shouldn't hurt anyway 🤷‍♂

Reply:
for some reason, when I set it up like this, the validation fails:
```
m6_statsd:
    servers:
        default:
            address: '%env(STATSD_SERVER)%'
            port:    '%env(int:STATSD_PORT)%'
    clients:
        default:
            servers: ['default']
```
even tho those are setup correctly with `udp://` in my env